crypto/mock/cacherMock.go

```diff
@@ -0,0 +1,146 @@
+package mock
```

Comment:
you can take this from testcommon package. like use it. no need to redefine

Reply:
Done.